docs/can-guides/commitment/migrating_to_3.0.md

```diff
@@ -1,54 +1,23 @@
 @page migrate-3 Migrating to CanJS 3
 @parent guides/commitment 2
 @templateRender <% %>
+@outline 0
```

Comment:
My guess is that it's trying to provide its own.  Also, there's no description (which show up if you hover over a link to this page).  I'm not sure it's worth breaking this convention.